ohara-manager/client/src/api/utils/__tests__/url.test.js

```diff
@@ -21,6 +21,10 @@ describe('toQueryParameters()', () => {
     expect(toQueryParameters()).toEqual('');
   });
 
+  it('returns "" if the params is give empty object', () => {
```

Comment:
Would probably rephrase this to:

```suggestion
  it('returns an empty string if the given params is an empty object', () => {
```